app/assets/stylesheets/application.scss

```diff
@@ -68,7 +68,13 @@ Default layout
   // noop
 }
 
-
+/*
+Banner when outside hero
+-----------------------------------------------
+*/
+.assessment-result-banner {
+  margin-bottom: govuk-spacing(7);
```

Comment:
Change to margin-bottom: govuk-spacing(8);

Reply:
This has now been updated to 8. I have also updated where the styling is. Let me know if there are any spacing updates that you would like here

Reply:
Also, regarding your comment above: the success banner above does not make use of this updated styling (where I have added the `#assessment-results` id). It has some separate styling for the success banner shown in the screenshot above

Reply:
@instantrick the post login success banner you screenshot as seen from the `My modules` page hasn't yet been changed. @martikat has a pattern that can apply banner styling globally or on specific pages. Is the banner's lower margin good on every other page, and if so, what value would you like to have used for the login banner?